include/libopencm3/stm32/g0/adc.h

```diff
@@ -64,48 +64,69 @@
 
 /** ADC_CALFACT Calibration factor register */
 #define ADC_CALFACT(adc)	MMIO32((adc) + 0xB4)
-
-/** ADC_OR Option register */
-#define ADC_OR(adc)			MMIO32((adc) + 0xD0)
 /**@}*/
 
+/* --- Register definitions -------------------------------------------------------*/
+
+#define ADC_SMPR(adc)		ADC_SMPR1(adc)
+#define ADC_TR(adc)		ADC_TR1(adc)
+
+#define ADC1_ISR		ADC_ISR(ADC1)
+#define ADC1_IER		ADC_IER(ADC1)
+#define ADC1_CR			ADC_CR(ADC1)
+#define ADC1_CFGR1		ADC_CFGR1(ADC1)
+#define ADC1_CFGR2		ADC_CFGR2(ADC1)
+#define ADC1_SMPR		ADC_SMPR(ADC1)
+#define ADC1_TR			ADC_TR(ADC1)
+#define ADC1_CHSELR		ADC_CHSELR(ADC1)
+#define ADC1_DR			ADC_DR(ADC1)
+#define ADC1_CCR		ADC_CCR(ADC1)
+
+#define ADC1_AWD1TR		ADC_AWD1TR(ADC1)
+#define ADC1_AWD2TR		ADC_AWD2TR(ADC1)
+#define ADC1_AWD3TR		ADC_AWD3TR(ADC1)
+#define ADC1_AWD2CR		ADC_AWD2CR(ADC1)
+#define ADC1_AWD3CR		ADC_AWD3CR(ADC1)
+#define ADC1_CALFACT		ADC_CALFACT(ADC1)
+
 /* --- Register values -------------------------------------------------------*/
 
 /** @addtogroup adc_isr
 @{*/
 
 /** CCRDY: Channel Configuration Ready flag */
-#define ADC_ISR_CCRDY				(1 << 13)
+#define ADC_ISR_CCRDY		(1 << 13)
 
 /**@}*/
 
 /** @addtogroup adc_ier
 @{*/
 
 /** CCRDYIE: Channel Configuration Ready Interrupt enable bit */
-#define ADC_IER_CCRDYIE				(1 << 13)
+#define ADC_IER_CCRDYIE		(1 << 13)
 
 /**@}*/
 
 /** @addtogroup adc_ccr
 @{*/
 
-#define ADC_CCR_PRESC_MASK		(0xf)
-#define ADC_CCR_PRESC_SHIFT		(18)
+#define ADC_CCR_PRESC_SHIFT	18
+#define ADC_CCR_PRESC_MASK	(0xf << ADC_CCR_PRESC_SHIFT)
```

Comment:
this is going _back_ to shifted style, which we're trying not to do.

Reply:
I'm starting to lose my track... This is a `_MASK` field, not a enum value. It's shifted everywhere within libopencm3. What's the point of having unshifted `_MASK` defines?

Reply:
So, do I get it right that `*_MASK` should be unshifted and `*_VAL` stay shifted?